stub-idp/src/integration-test/java/uk/gov/ida/apprule/ServiceListServiceIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.ida.apprule;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import io.dropwizard.client.JerseyClientConfiguration;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.util.Duration;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import uk.gov.ida.apprule.support.IntegrationTestHelper;
+import uk.gov.ida.apprule.support.StubIdpAppRule;
+import uk.gov.ida.jerseyclient.ErrorHandlingClient;
+import uk.gov.ida.jerseyclient.JsonClient;
+import uk.gov.ida.jerseyclient.JsonResponseProcessor;
+import uk.gov.ida.restclient.ClientProvider;
+import uk.gov.ida.stub.idp.configuration.SingleIdpConfiguration;
+import uk.gov.ida.stub.idp.domain.Service;
+import uk.gov.ida.stub.idp.exceptions.FeatureNotEnabledException;
+import uk.gov.ida.stub.idp.services.ServiceListService;
+
+import javax.inject.Inject;
+import java.util.Optional;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ServiceListServiceIntegrationTest extends IntegrationTestHelper {
```

Comment:
This is probably a bit confusing but our integration tests usually test the app end to end (rather than testing a single service). Perhaps this test could be ported to a unit test?